### DISCUSSION TITLE
RESULT_CODE_KILLED_BAD_MESSAGE

### DISCUSSION THREAD

**Discussion by vinaykapoor** [MicrosoftEdge/WebView2Feedback#3912] [Q&A]
I am using webview2 with WPF on .NET Core 5. In one of the users PC, he gets this message 
Cant open this page
Restart Microsoft Edge
Restart your Computer
Error Code: RESULT_CODE_KILLED_BAD_MESSAGE.
I am not able to figure out why that is.

**Comment by champnic** [comment]
Hey @vinaykapoor - Can you add some more details here? What component is returning this error code? What page can't be opened?

**Comment by vinaykapoor** [comment]
I am attempting to use webview2 for WPF on .net core 5.0. I followed the steps mentioned in the documentation, namely install the webview2 runtime. I was able to do that and was able to add webview2 to my wpf application. It worked well on most users PC. However on some users PC the webview2 component doesn’t work and throws that message, despite installing the runtime component. 

Vinay

**Comment by champnic** [comment]
Is there a stack trace or a particular function call that's returning this code?

**Comment by vinaykapoor** [comment]
I followed steps from https://docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/wpf
Microsoft Edge
Version 93.0.961.47 (Official build) (64-bit)
I used the Webview 2 standalone installer.
https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section
 
For most users the chromium window shows up but for others I get the
Error Code: RESULT_CODE_KILLED_BAD_MESSAGE
 
Can you give me any suggestions on how to try to debug this on their PC.

Vinay

**Comment by champnic** [comment]
Could you share a screenshot of where you are seeing this error code?

If you want to debug this the best is to have a debugger attached (or remotely) using Visual Studio, and getting a stack of what's generating this error code.

**Comment by vinaykapoor** [comment]
All I am doing is setting the source for the webview control to a local
intranet site. I dont get any exception but i the page displays exactly
that comment.

**Comment by champnic** [comment]
Do you get this error if you navigate to that page in the Microsoft Edge browser on the machines that are running into this?

**Comment by vinaykapoor** [comment]
No when I try to go directly it’s fine. 

Vinay

**Comment by vinaykapoor** [comment]
I can try to uninstall and reinstall to see if it makes a difference. I am running on user machines who do not have elevated rights so the only way to install is using a temporary account to install . 

Vinay

**Comment by vinaykapoor** [comment]
I am attempting to install this for users with different types of rights on their PC. That’s probably why it works for some and not others. I am not getting any exception but I can see that it’s because of their roles . Can you explain what access does a person need to be able to access webview . 

Vinay

**Comment by vinaykapoor** [comment]
I tried to run your wpf sample. I get the following exception. 



Vinay

**Comment by vinaykapoor** [comment]
![2F22DAAC-D9DC-4605-876F-2F3D09445344](https://user-images.githubusercontent.com/923513/136829096-ad6c83e5-4cb2-4b16-b87c-ea79bc080026.png)

**Comment by champnic** [comment]
Can you check the <user data folder>\EBWebView\Crashpad\reports for a crash dump?

Elevation is needed to install the WebView2 runtime, but once installed it shouldn't make a difference. It installs to the system (why it currently needs elevation) and every user of the device should be able to use it. If there was an access problem you wouldn't see the WebView2 load at all. I'm not sure if the access differences could be causing the page they are trying to access to fail to load.
